pallets/offchain-demo/Cargo.toml

```diff
@@ -9,15 +9,15 @@ license = "GPL-3.0-or-later"
 
 [dependencies]
 # external dependencies
-codec = { package = "parity-scale-codec", version = "1.3.0", default-features = false }
+parity-scale-codec = { default-features = false, features = ['derive'], version = '1.3.0' }
 parking_lot = "0.10.0"
-serde = { version = "1.0.101", optional = true }
-serde_json = { version = "1.0.46", default-features = false, features = ["alloc"] }
-simple-json2 = { version = "0.1.2", default-features = false }
+
+alt_serde = { version = "1", default-features = false, features = ["derive"] }
+serde_json = { version = "1", default-features = false, git = "https://github.com/Xanewok/json", branch = "no-std", features = ["alloc"] }
```

Comment:
This should be `alt_serde_json` from crates.io, when it's released. Maybe add a comment here to not forget it.